src/CachePoolTest.php

```diff
@@ -316,22 +316,6 @@ public function testSave()
         $this->assertEquals('value', $this->cache->getItem('key')->get());
     }
 
-    public function testSaveWithNoData()
-    {
-        if (isset($this->skippedTests[__FUNCTION__])) {
-            $this->markTestSkipped($this->skippedTests[__FUNCTION__]);
-
-            return;
-        }
-
-        $item   = $this->cache->getItem('key');
-        $return = $this->cache->save($item);
-
-        $this->assertFalse($return, 'When saving an object with no data, we should generate an error.');
```

Comment:
Discussion about this started here: https://github.com/symfony/symfony/pull/17408#issuecomment-172456817